runtime/runtime/src/prefetch.rs

```diff
@@ -304,6 +328,73 @@ impl TriePrefetcher {
         Ok(())
     }
 
+    /// Prefetcher tuned for claim.sweat::claim contract calls.
+    ///
+    /// Remove after #10965 reaches mainnet.
+    fn prefetch_claim_sweat_claim(
+        &self,
+        account_id: AccountId,
+        predecessor: AccountId,
+    ) -> Result<(), PrefetchError> {
+        let Self { prefetch_api, trie_root, .. } = self;
+        let trie_root = *trie_root;
+        let prefetch_api = prefetch_api.clone();
+        rayon::spawn(move || {
+            let mut account_data_key = Vec::with_capacity(4 + 8 + predecessor.len());
+            let Ok(()) = 0u8.serialize(&mut account_data_key) else { return };
+            let Ok(()) = predecessor.serialize(&mut account_data_key) else { return };
+            let trie_key =
+                TrieKey::ContractData { account_id: account_id.clone(), key: account_data_key };
+            // Just read this directly for now since this is temporary anyway
+            let prefetcher_storage = prefetch_api.make_storage();
+            let trie = Trie::new(prefetcher_storage, trie_root, None);
+            let Ok(Some(account_record)) = trie.get(&trie_key.to_vec()) else {
+                tracing::debug!(
+                    target: "runtime::prefetch",
+                    message = "could not load AccountRecord",
+                    key = ?trie_key,
+                );
+                return;
+            };
+            #[derive(borsh::BorshDeserialize)]
+            #[allow(dead_code)]
+            struct AccountRecord {
+                accruals: Vec<(u32, u32)>,
+                is_enabled: bool,
+                claim_period_refreshed_at: u32,
+                is_locked: bool,
+            }
+            let Ok(account_record) = borsh::from_slice::<AccountRecord>(&account_record) else {
+                tracing::debug!(
+                    target: "runtime::prefetch",
+                    message = "could not decode AccountRecord",
+                );
+                return;
+            };
+
+            for (dt, idx) in account_record.accruals {
+                let mut accruals_key = Vec::with_capacity(4 + 8);
+                // StorageKey::Accruals
+                let Ok(()) = 1u8.serialize(&mut accruals_key) else { continue };
+                let Ok(()) = dt.serialize(&mut accruals_key) else { continue };
+                let accruals_key = sha2::Sha256::digest(&accruals_key).to_vec();
+                let _ = prefetch_api.prefetch_trie_key(
```

Comment:
Going through `prefetch_api.prefetch_trie_key` puts this request [at the end](https://github.com/near/nearcore/blob/0efb5f71d138503f64625676d6b46cdfdcaa4970/core/store/src/trie/prefetching_trie_storage.rs#L443) of the prefetcher's FIFO work queue.

Bypassing the queue might be avoided by using `trie.get` as above, to have the data available earlier. However bypassing the queue and prefetcher API like that for many keys is probably risky.